fic_modules/markdown_modules.py

```diff
@@ -106,7 +106,8 @@ def create_git_md_table(repository_name, path_to_files):
         md_file.close()
     except FileNotFoundError:
         if LOGGER:
-            print("Json for {} is empty! Skipping!".format(repository_name))
+            LOGGER.info("Json for {} is empty! Skipping!".format(repository_name))
+            # print("Json for {} is empty! Skipping!".format(repository_name))
```

Comment:
If we don't require this anymore, it should be removed.

Reply:
Fixed.